mods/d2k/rules/aircraft.yaml

```diff
@@ -108,6 +108,13 @@ ornithopter:
 		TurnSpeed: 8
 		Repulsable: False
 		CruiseAltitude: 1920
+		AirborneCondition: airborne
+	Targetable@GROUND:
+		TargetTypes: Ground, Vehicle
+		RequiresCondition: !airborne
+	Targetable@AIRBORNE:
+		TargetTypes: Air
+		RequiresCondition: airborne
```

Comment:
Those never land as they are only called in via support powers. So the `@Ground` targetable should not be needed. Can we just give them
```
Targetable:
	TargetTypes: Air
```
?